Documentation/security/network/encryption-ipsec.rst

```diff
@@ -166,7 +166,7 @@ To replace cilium-ipsec-keys secret with a new key:
 
 .. code-block:: shell-session
 
-    KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o yaml | awk '/^\s*keys:/ {print $2}' | base64 -d | awk '{print $1}')
+    KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o go-template --template={{.data.keys}} | base64 -d | cut -c 1)
     if [[ $KEYID -ge 15 ]]; then KEYID=0; fi
     data=$(echo "{\"stringData\":{\"keys\":\"$((($KEYID+1))) "rfc4106\(gcm\(aes\)\)" $(echo $(dd if=/dev/urandom count=20 bs=1 2> /dev/null| xxd -p -c 64)) 128\"}}")
     kubectl patch secret -n kube-system cilium-ipsec-keys -p="${data}" -v=1
```

Comment:
Maybe out-of-scope for this PR, but I'm wondering if we could add something in the list of commands to fail early if a command failed? Like some check on the `KEYID` or on `data` (length check?)?

Reply:
Don't know if it's a good idea, but how about making it a `cilium` sub-command, such as `cilium encrypt key-rotation`?

Reply:
I think a cilium encrypt subcommand to obtain the key information and another to do the complete key rotation would be quite valuable (but out of scope for this PR).